pyproject.toml

```diff
@@ -26,14 +26,14 @@ classifiers = [
   'Programming Language :: Python :: 3.12',
 ]
 dependencies = [
-  'zarr>=2.5.0',
-  'numcodecs>=0.5.7',
-  'scipy>=1.2.1',
+  'zarr>=2.16.0',
+  'numcodecs>=0.12.0',
+  'scipy>=1.9.3',
   'negspy>=0.2.24',
   'pandas>=1.1.2',
```

Comment:
Does `pandas` also need to be updated? It looks like there's a new major version out as of last year: https://pandas.pydata.org/docs/whatsnew/index.html

CC @keller-mark

Reply:
@NickAkhmetov thanks for reviewing. I can change it, but there are tests for Python3.8 and Python3.12, so pandas has to be compatible with Python3.8. 
Out of curiosity, wouldn't the latest package be downloaded based on the python version?
I have not changed the others' either, as the restrictions seemed fine. can bump those as well (where possible)

Reply:
Thanks for the explanation. makes sense

Reply:
Actually - on further reflection, I was overthinking it. If the numpy version is `>1.21.1`, we can actually use the previous pandas version number, since the package manager should be able to resolve the appropriate pairing of numpy/pandas versions that should be used in a given circumstance. Apologies for the confusion 😅